bin/cron/applab_datasets/weather

```diff
@@ -165,6 +197,7 @@ end
 def main
   fb = FirebaseHelper.new('shared')
   records, columns = get_weather_data
+  fb.delete_shared_table('weather')
   fb.upload_shared_table('weather', records, columns)
```

Comment:
Is there a possibility that students querying the weather table will get nothing because it's been deleted but not re-uploaded? Is this a problem?

Reply:
If someone happens to be looking at the table when the script is run (once per day), then yes I think they will see nothing for a fraction of a second. But then the new data will pop in automatically when the new table is uploaded. To me that doesn't seem like a big deal, especially for v1, but what do you think?

Reply:
✔️  ^ I agree. Since it auto-populates, it's not a big deal.

Reply:
✔️ ^ I agree. Since it auto-populates, it's not a big deal.